docs/component/rl/quickstart.rst

```diff
@@ -0,0 +1,174 @@
+
+    order_file: ./data/backtest_orders.csv
+    start_time: "9:45"
+    end_time: "14:44"
+    qlib:
+        provider_uri_1min: ./data/bin
+        feature_root_dir: ./data/pickle
+        # feature generated by today's information
+        feature_columns_today: [
+            "$open", "$high", "$low", "$close", "$vwap", "$volume",
+        ]
+        # feature generated by yesterday's information
+        feature_columns_yesterday: [
+            "$open_v1", "$high_v1", "$low_v1", "$close_v1", "$vwap_v1", "$volume_v1",
+        ]
+    exchange:
+        # the expression for buying and selling stock limitation
+        limit_threshold: ['$close == 0', '$close == 0']
+        # deal price for buying and selling
+        deal_price: ["If($close == 0, $vwap, $close)", "If($close == 0, $vwap, $close)"]
+    volume_threshold:
+        # volume limits are both buying and selling, "cum" means that this is a cumulative value over time
+        all: ["cum", "0.2 * DayCumsum($volume, '9:45', '14:44')"]
+        # the volume limits of buying
+        buy: ["current", "$close"]
+        # the volume limits of selling, "current" means that this is a real-time value and will not accumulate over time
+        sell: ["current", "$close"]
+    strategies: 
+        30min: 
+            class: TWAPStrategy
+            module_path: qlib.contrib.strategy.rule_strategy
+            kwargs: {}
+        1day: 
+            class: SAOEIntStrategy
+            module_path: qlib.rl.order_execution.strategy
+            kwargs:
+            state_interpreter:
+                class: FullHistoryStateInterpreter
+                module_path: qlib.rl.order_execution.interpreter
+                kwargs:
+                max_step: 8
+                data_ticks: 240
+                data_dim: 6
+                processed_data_provider:
+                    class: PickleProcessedDataProvider
+                    module_path: qlib.rl.data.pickle_styled
+                    kwargs:
+                    data_dir: ./data/pickle_dataframe/feature
+            action_interpreter: 
+                class: CategoricalActionInterpreter
+                module_path: qlib.rl.order_execution.interpreter
+                kwargs: 
+                values: 14
+                max_step: 8
+            network: 
+                class: Recurrent
+                module_path: qlib.rl.order_execution.network
+                kwargs: {}
+            policy: 
+                class: PPO
+                module_path: qlib.rl.order_execution.policy
+                kwargs: 
+                    lr: 1.0e-4
+                    # the path for the latest model in the training process
+                    weight_file: ./checkpoints/latest.pth
+    # Concurrent environment workers.
+    concurrency: 5
+
+With the above config files, you can start training the agent by the following command:
+
+.. code-block:: console
+
+    $ python qlib/rl/contrib/train_onpolicy.py --config_path train_config.yml
```

Comment:
Use `python -m qlib.rl.contrib.train_onpolicy`. Otherwise users must clone qlib to run this.

Reply:
fixed